### ISSUE TITLE
How can I predict a single wav file against this model？

### ISSUE THREAD

**Issue by tongjiahui** [keras-team/keras-io#1319]
How can I predict a single wav file against this model？

**Comment by dhantule** [comment]
Hi @tongjiahui, thanks for the report. 

To predict on a particular `.wav` file you'll have to select the specific `.wav ` file you want to infer, preprocess it (apply the same transformations you used during training), and then feed it into the model.

You'll need to load the audio file, convert it to a spectrogram, and normalize it just as you did during the training.

Please refer this[ tutorial](https://www.tensorflow.org/tutorials/audio/simple_audio) that'll help you run inference on `.wav `files.
